interface/modules/custom_modules/oe-module-faxsms/rcauth.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/**
+ * Fax SMS Module Member
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2018-2021 Jerry Padgett <sjpadgett@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+$ignoreAuth = 1;
+$sessionAllowWrite = true;
+require_once(__DIR__ . "/../../../globals.php");
+
+use OpenEMR\Common\Crypto\CryptoGen;
+use RingCentral\SDK\SDK;
+
+$url = $_SESSION['url'];
+$callbackUrl = $_SESSION['redirect_uri'];
+
+function processCode(): void
+{
+    $vendor = '_ringcentral';
+    $cacheDir = $GLOBALS['OE_SITE_DIR'] . '/documents/logs_and_misc/_cache';
+    $authUser = 0;
+    $credentials = sqlQuery("SELECT * FROM `module_faxsms_credentials` WHERE `auth_user` = ? AND `vendor` = ?", array($authUser, $vendor))['credentials'];
+    if (empty($credentials)) {
+        echo xlt("Applications credentials were not found. Please setup account.");
+        die('Credential Error');
+    }
+    $cryptoGen = new CryptoGen();
+    $credentials = json_decode($cryptoGen->decryptStandard($credentials), true, 512, JSON_THROW_ON_ERROR);
```

Comment:
this is correct use of decryptStandard since you are using the key from the drive to decrypt something from the database